front/src/components/atoms/CustomButton/CustomButton.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import CustomButton from './CustomButton';
+
+export default {
+  title: 'Buttons/CustomButton',
```

Comment:
```suggestion
  title: 'Atom/Buttons',
```

스토리보드도 Atomic design처럼 구분해두었으면 좋겠어요.